x-pack/plugins/cases/common/schema/index.ts

```diff
@@ -154,6 +157,33 @@ export const limitedNumberSchema = ({ fieldName, min, max }: LimitedSchemaType)
     rt.identity
   );
 
+export const limitedJSNumberSchema = ({ fieldName }: LimitedJSNumberSchemaType) =>
+  new rt.Type<number, number, unknown>(
+    'LimitedNumber',
+    rt.number.is,
+    (input, context) =>
+      either.chain(rt.number.validate(input, context), (s) => {
+        if (Math.abs(s) > Number.MAX_VALUE) {
```

Comment:
I think we might have to rethink this limit.

The mapping for the `number` custom fields is [long](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/saved_object_types/cases/cases.ts#L209), this is [smaller](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_VALUE) than `Number.MAX_VALUE`.

(The same applies to `Number.MIN_VALUE`)

CC: @cnasikas

Reply:
You are right @adcoelho! I told @guskovaue to use `Number.MAX_VALUE` but I missed that ES has a limit. We should update to the limits of the `long` ES field type.